app/models/course.rb

```diff
@@ -8,4 +8,12 @@ class Course < ApplicationRecord
   validates :cuisine_type, presence: true
   validates :duration, presence: true
   validates :price, presence: true
+
+  def self.search(keywords)
+    if keywords
+      where("name LIKE ?", "%#{keywords}%").order('id DESC')
```

Comment:
Use `ILIKE` here instead of `LIKE` to ignore case when you are matching values.

Search will not match "Sushi" with `LIKE`:
<img width="1076" alt="Screen Shot 2021-08-17 at 23 09 59" src="https://user-images.githubusercontent.com/31387233/129741435-cf2a504d-032c-4fd8-a0c0-f0d352397ecc.png">

Using `ILIKE` will match properly:
<img width="1479" alt="Screen Shot 2021-08-17 at 23 11 47" src="https://user-images.githubusercontent.com/31387233/129741759-64b7a5dc-03fa-49c7-8c2f-f8fa23ae008c.png">

```suggestion
      where("name ILIKE ?", "%#{keywords}%").order('id DESC')
```